### DISCUSSION TITLE
Backreferences: titles vs preview

### DISCUSSION THREAD

**Discussion by WhyNotHugo** [zk-org/zk#456] [General]
Using neovim, I have two ways of accessing notes. The first is the generic way to find references via an LSP: `require("fzf-lua").lsp_references()`.

This works, and shows me a file picker. Moving through the file picker, I see a preview of the referencing note. However, the picker itself shows filenames, so I don't see the titles of each entry.

The second way to access backreferences is with `:ZkBacklinks`. This shows a picker with the titles of the referencing notes, but not the preview with the context of where the reference happens.

Can I somehow get both; titles in the picker and previews?

**Comment by tjex** [comment]
I'm not sure with fzf, but with telescope there is previews alongside the result titles. I would've thought this should work out of the box, as the code in `zk-nvim` lua simply swaps in and out pickers, but applies the same display logic (from memory). Maybe the picker config requires some extra configuring either in .zk/config.toml or in neovim? (e.g with what to display `bat`, `cat`, etc, like in regular `fzf` config)
